packages/typespec-test/test/contentsafety_modular/generated/typespec-ts/review/ai-content-safety.api.md

```diff
@@ -9,6 +9,16 @@ import { KeyCredential } from '@azure/core-auth';
 import { OperationOptions } from '@azure-rest/core-client';
 import { TokenCredential } from '@azure/core-auth';
 
+// @public
+export interface AddOrUpdateBlockItemsOptions {
+    blockItems: TextBlockItemInfo[];
+}
+
+// @public (undocumented)
+export interface AddOrUpdateBlockItemsOptions {
```

Comment:
Seems to have duplicate exports here for 'AddOrUpdateBlockItemsOptions', is this expected?

Reply:
Let me double check on this.